messaging/src/main/java/org/axonframework/commandhandling/gateway/IntervalRetryScheduler.java

```diff
@@ -57,10 +51,17 @@
      * @param builder the {@link Builder} used to instantiate a {@link IntervalRetryScheduler} instance
      */
     protected IntervalRetryScheduler(Builder builder) {
+        super(builder);
+
         builder.validate();
         this.retryInterval = builder.retryInterval;
-        this.maxRetryCount = builder.maxRetryCount;
-        this.retryExecutor = builder.retryExecutor;
+    }
+
+    @Override
+    protected long computeRetryInterval(CommandMessage commandMessage,
```

Comment:
Two minor things.
1. Indenting is slightly off here.
2. Shouldn't the type of the second parameter be `Throwable` (or `Exception`) instead of `RuntimeException`? :thinking:

Reply:
See previous comment on same issue.

Reply:
Allard used his overrule powers (with fair reasoning), so I'll resolve this comment for you.